demo/addons/fmod/tool/property_editors/FmodParameterUi.gd

```diff
@@ -0,0 +1,28 @@
+@tool class_name FmodParameterUi extends VBoxContainer
+
+static var _step := 0.1
+
+func _ready():
+	%Slider.value_changed.connect(_on_slider_value_change)
```

Comment:
same here - we should always cache nodes via `@onready var` for better performance.

Reply:
dead code, will remove